docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
+  mysql:
+    image: my-sql:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql-container
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
```

Comment:
These envs are set up on the Dockerfile of MySQL, no need to define it one more time

Reply:
Done

Reply:
@NazarSheremet 
But why do we keep the ENV sets for the phytonapp part? They also already exist in the Dockerfile.

Reply:
Not quite so, in this task we do not define environment variables for connecting to the database in Docker and hardcode them inside the Python code

Reply:
Got it, thanks